public/locales/en/common.json

```
@@ -9,11 +11,14 @@
   "developerTraits": "Developer traits",
   "failedLoadTraits": "Failed to load traits",
   "madeBy": "Made by the $t(title) community: ",
-  "owner": "Owner",
+  "daoGithubOrg": "Developer DAO GitHub organization",
+  "daoGithubRepo": "Developer DAO GitHub repository",
+  "owner": "Owner:",
   "unclaimed": "Unclaimed",
   "enterDeveloperId": "Please enter a $t(title) id",
   "hosting": "Hosting by:",
   "copyLinkToNFT": "Copy link to NFT",
   "linkCopied": "Link copied to clipboard",
-  "projects": "A list of community projects created by the Developer DAO community."
+  "projectsList": "A list of community projects created by the Developer DAO community.",
```

```suggestion
  "projectsList": "A list of community projects created by the $(title) community.",
```

Also has me wondering if we should translate "community" separately as well, since that's a fairly common word.